### PR TITLE
replaced with arrow function

### PR DESCRIPTION
**Pull request by sendilkumarn**

### CODE REVIEW

File: src/directive/show-validation.directive.ts [hunk 1]
```diff
@@ -19,22 +19,22 @@ import { Directive, ElementRef, Renderer } from '@angular/core';
 export class ShowValidationDirective {
     constructor(el: ElementRef, renderer: Renderer) {
        let formElements = el.nativeElement.querySelectorAll('.form-group');
-       Array.prototype.forEach.call(formElements, (formGroup: any) => {
-           // TODO this needs to be properly tested
+       formElements.forEach((formGroup) =>  {
```
Comment (deepu105) [inline src/directive/show-validation.directive.ts]: @sendilkumarn did this work? coz formElements is not a proper array its array like

File: src/directive/show-validation.directive.ts [hunk 2]
```diff
@@ -19,22 +19,22 @@ import { Directive, ElementRef, Renderer } from '@angular/core';
...
            let inputs = formGroup.querySelectorAll('input[(ngModel)],textarea[(ngModel)],select[(ngModel)]');
 
-           if (inputs !== null) {
-               Array.prototype.forEach.call(inputs, (input: any) => {
-                   // TODO this logic needs to be checked and fixed accordingly
-                   let isInvalid = input.classList.contains('ng-invalid') && input.classList.contains('ng-dirty');
-                   formGroup.classList.toggle('has-error', isInvalid);
-                   /*scope.$watch(function() {
+           if (inputs) {
+               inputs.forEach((input) => {
```
Comment (deepu105) [inline src/directive/show-validation.directive.ts]: same here

**Comment by sendilkumarn** [comment]
checked it ✅

File: package.json [hunk 3]
```diff
@@ -75,5 +75,5 @@
     "webpack": "2.2.0-rc.2",
     "zone.js": "0.7.4"
   },
-  "version": "0.0.13"
+  "version": "0.0.14"
```
Comment (deepu105) [inline package.json]: can you not bump this please as the release scripts will do it automatically. else next release will become 0.15 instead of 0.14

**Pull request merged**